modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -83,18 +83,17 @@ exports.signout = function (req, res) {
 /**
  * OAuth provider call
  */
-exports.oauthCall = function (strategy, scope) {
-  return function (req, res, next) {
+exports.oauthCall = function (req, res, next) {
+    var strategy = req.params.strategy;
     // Authenticate
-    passport.authenticate(strategy, scope)(req, res, next);
+    passport.authenticate(strategy)(req, res, next);
```

Comment:
@Wuntenn No problem at all. I've also been working on getting the Azure-AD passport module integrated, which is when I read the Passport documentation and found out Scope was valid in the passport.use declaration. I wasn't positive about how the null would be handled on the passport.authenticate until I tested it though.

BTW, I should have another Pull request here soon with the passport-azure-ad module integrated. I've got that stuff done, but submitted a pull request to passport-azure-ad that I'm hoping they'll integrate first. I also wanted to see what happened with the two Pull requests I did here before sending another pull request, since it would change the number of files I'd have to modify.